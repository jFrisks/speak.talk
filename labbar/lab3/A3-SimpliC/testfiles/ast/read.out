Program
  List
    FunctionDecl
      IdDecl ID="a"
      List
      Block
        List
          IdDeclStmt
            IdDecl ID="x"
          Assignment
            IdUse ID="x"
            FunctionCall ID="read"
              List
