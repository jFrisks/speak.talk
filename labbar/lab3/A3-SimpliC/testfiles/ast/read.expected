Program
  List
    FunctionDecl
      IdDecl ID="a"
      List
      Block
        List
          IdDeclStmt
            IdDecl ID="x"
          Assignment
            IdUse ID="x"
            FunctionCall
              IdUse ID="read"
              List